src/_component/open-source/open-source.pug

```diff
@@ -39,7 +39,15 @@
     .github-project__title=githubProjects
     .space(height="10")
     .github-project
+        input.project-input(type="radio" checked name="checkbox-input" id="default-project__input")
         .project-1
+            .icons
+                label.project-description__label(for="project-input-id")
+                    .bottom-icon__info.icon-info
+                label.project-description__close(for="default-project__input")
+                    .bottom-icon__cross.icon-cross
+                a.bottom-icon__youtube(href="https://www.youtube.com/" class=`icon-youtube`)
```

Comment:
you have double quotes for the first attribute, ticks for the second one.
This is called **inconsistency** and an employer might see this as a disadvantage, more of you don't know very well what's what or what you are doing.